### DISCUSSION TITLE
Attribute request for Engineer Wrench

### DISCUSSION THREAD

**Discussion by higps** [nosoop/SM-TFCustomAttributeStarterPack#62] [Q&A]
I've been trying to get the ability to have the engineer have the same behaviour as the TF_bot engineers where they can phase through their own buildings while still being able to hit the wrench.

I would like to submit a request to have this made as an attribute, as the closest I've been able to get is to have the collision off, but that makes it so the wrench doesn't hit.

How it would work:
1. Collision Disabled on dispensers and sentries
2. Wrench has to hit while the collision is off

**Comment by higps** [comment]
It would basically function the same as if the building belonged to a teammate

**Comment by nosoop** [comment]
Hi, thanks for opening an inquiry.  If this is a paid request, do [contact me via the AlliedModders forum PM system](https://forums.alliedmods.net/member.php?u=252787) for further discussion.
